specification/specs/cluster/cluster_reroute/ClusterRerouteRequest.ts

```diff
@@ -23,7 +23,7 @@ class ClusterRerouteRequest extends RequestBase {
     dry_run?: boolean
     explain?: boolean
     master_timeout?: Time
-    metric?: string[]
+    metric?: string | string[]
```

Comment:
We have a `Metrics` alias for this :)
```suggestion
    metric?: Metrics
```

Reply:
added